src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/common/TestingUtils.java

```diff
@@ -1,24 +1,23 @@
 package uk.gov.hmcts.futurehearings.hmi.acceptance.common;
 
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.util.ResourceUtils;
+
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
-import io.restassured.response.Response;
-import lombok.extern.slf4j.Slf4j;
-import org.json.JSONException;
-import org.skyscreamer.jsonassert.JSONAssert;
-import org.skyscreamer.jsonassert.JSONCompareMode;
-import org.springframework.util.ResourceUtils;
-
 @Slf4j
 public class TestingUtils {
```

Comment:
class can be declared as final class, as this class is having private constructor and no other class is extending this Util class

Reply:
Out of scope